src/main/docker/root/docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [[ ! -d "$PGDATA" ]]
+then
+  mkdir -p "$PGDATA"
+fi
+
+if [[ $(stat -c %U "$PGDATA") != 'postgres' ]]
+then
+  chown -R postgres:postgres "$PGDATA"
+fi
+
+if [[ $(stat -c %a "$PGDATA") != '700' ]]
+then
+  chmod 700 "$PGDATA"
+fi
+
+mkdir --parents /root/.config/patroni
+ln -s /config/patronictl.yaml /root/.config/patroni/patronictl.yaml
+
+/usr/bin/timescaledb-tune --quiet --yes --conf-path=$PGDATA --dry-run
```

Comment:
this command will only output the recommended pg settings but not apply them, see
https://docs.timescale.com/timescaledb/latest/how-to-guides/configuration/timescaledb-tune/

perhaps write the recommended pg settings into a new conf file in /config/patroni?

Reply:
was a leftover from testing
timescaledb-tune needs a postgres config as input, so should run after patroni has created one
but patroni only uses these settings when initially creating the database I think